kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/strimzi/KafkaUserType.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.resources.strimzi;
+
+import java.util.function.Consumer;
+
+import io.fabric8.kubernetes.api.model.DeletionPropagation;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.skodjob.testframe.interfaces.ResourceType;
+import io.strimzi.api.kafka.model.user.KafkaUser;
+import io.strimzi.api.kafka.model.user.KafkaUserList;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.k8s.KubeClusterResource;
+
+/**
+ * The type Kafka user resource.
+ */
+public class KafkaUserType implements ResourceType<KafkaUser> {
```

Comment:
Do we actually use this KafkaUser?  YAGNI...

Reply:
Legacy code that is not used any more. Thanks!